int32 id
int32 history
int32 measured
int32 type
int32 age
float32 x1
float32 y1
float32 x2
float32 y2
float32 zMin
float32 zMax
float32 vX
float32 vY
float32 vZ
float32 aX
float32 aY
float32 aZ
float32 existenceProbability
float32 vXQuality
float32 vYQuality
float32 distanceToEgo
